newsroom/wire/views.py

```diff
@@ -173,6 +174,10 @@ def bookmarks():
 
 @blueprint.route('/wire/search')
 def search():
+    if app.config['PREPEND_EMBARGOED_TO_WIRE_SEARCH']:
+        args = request.args.to_dict()
+        args['prepend_embargoed'] = True
```

Comment:
maybe we could use here `args.setdefault('prepend_embargoed', True)` so we could control it from UI if needed